src/services/impl/write/UserService.js

```diff
@@ -0,0 +1,158 @@
+import BaseService from './../../base/BaseService';
+import UserSchema from './../../../schemas/UserSchema';
+import { USER_PROFILE_STATUS } from '../../../constants';
+import config from './../../../config';
+
+class UserService extends BaseService {
+
+  constructor(options = { scoped: true }) { 
+    super(UserSchema, options);
+  }
+
+  async createUserProfile({
```

Comment:
Let's remove the `Profile` postfix from all methods of UserService. Now we treat this doc as User entity similarly to team, Project, etc